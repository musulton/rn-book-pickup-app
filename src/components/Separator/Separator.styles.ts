import {StyleSheet, ViewStyle} from "react-native";

import {COLORS} from "../../constants";

interface Styles {
    separator: ViewStyle
}

const styles = StyleSheet.create<Styles>({
    separator: {
        height: 1,
        backgroundColor: COLORS.lightGray3
    }
})

export default styles
