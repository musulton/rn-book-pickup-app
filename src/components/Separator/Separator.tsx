import {View} from "react-native";
import styles from "./Separator.styles";

const Separator = () => {
    return <View style={styles.separator} />
}

export default Separator
