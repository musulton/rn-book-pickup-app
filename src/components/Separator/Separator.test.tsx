import renderer from "react-test-renderer"

import Separator from "./Separator";

describe("<Separator />", () => {
    it("should render Separator correctly", () => {
        const tree = renderer.create(<Separator />).toJSON()

        expect(tree).toMatchSnapshot();
    })
});
