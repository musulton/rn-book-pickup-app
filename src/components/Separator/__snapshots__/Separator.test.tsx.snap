// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> should render Separator correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#D4D5D6",
      "height": 1,
    }
  }
/>
`;
